FrontEndReact/src/View/Admin/View/ViewAssessmentTask/ViewAssessmentTasks.js

```diff
@@ -117,9 +126,9 @@ class ViewAssessmentTasks extends Component {
                     setCellProps: () => { return { width:"20px"} },
                     customBodyRender: (suggestions) => {
                         return(
-                            <p>
+                            <>
                                 {suggestions ? (suggestions ? "Yes" : "No") : "No"}
```

Comment:
`{suggestions ? (suggestions ? "Yes" : "No") : "No"}` -> `suggestions ? "Yes" : "No";`

Reply:
I committed changes to remove the outer ternary operator!